Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM atddocker/atd-oracle-py:production
+FROM --platform=linux/x86_64 atddocker/atd-oracle-py:production 
```

Comment:
not sure if this --platform arg is needed here? I needed it to build on my m1 mac ✨